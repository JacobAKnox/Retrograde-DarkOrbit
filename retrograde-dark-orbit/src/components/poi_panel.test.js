import { render } from '@testing-library/react'
import "setimmediate";
import POIPanel from './poi_panel';

it('chat_box component test', () => {
    const page = render(<POIPanel/>);
    expect(page).toMatchSnapshot();
});