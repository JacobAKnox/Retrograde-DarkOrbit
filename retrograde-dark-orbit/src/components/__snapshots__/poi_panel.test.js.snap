// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chat_box component test 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="text-xl text-white text-center items-center rounded-xl m-1 flex-grow"
      >
        <div
          class="bg-slate-900 rounded-xl m-1 p-2"
        >
          Action Points:
          <br />
          0
          /
          0
        </div>
        <div
          class="grid grid-cols-3 bg-slate-900 rounded-xl m-1 p-2"
        >
          <div
            class="rounded-xl p-2 m-2 bg-slate-700"
          >
            <div
              class="flex justify-center p-2 text-xl"
            >
              <p
                class="text-center text-wrap break-words"
              >
                name
              </p>
            </div>
            <div
              class="flex justify-center p-2"
            >
              <div
                class="flex rounded-xl m-1"
              >
                <div
                  class="flex p-1 rounded-l-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
                >
                  -
                </div>
                <div
                  class="flex bg-slate-900 p-1 min-w-12 justify-center select-none text-2xl"
                >
                  0
                </div>
                <div
                  class="flex p-1 rounded-r-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
                >
                  +
                </div>
              </div>
            </div>
            <div
              class="flex justify-center p-1 text-md"
            >
              <p
                class="text-center text-wrap break-words"
              />
            </div>
          </div>
          <div
            class="rounded-xl p-2 m-2 bg-slate-700"
          >
            <div
              class="flex justify-center p-2 text-xl"
            >
              <p
                class="text-center text-wrap break-words"
              >
                name1
              </p>
            </div>
            <div
              class="flex justify-center p-2"
            >
              <div
                class="flex rounded-xl m-1"
              >
                <div
                  class="flex p-1 rounded-l-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
                >
                  -
                </div>
                <div
                  class="flex bg-slate-900 p-1 min-w-12 justify-center select-none text-2xl"
                >
                  0
                </div>
                <div
                  class="flex p-1 rounded-r-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
                >
                  +
                </div>
              </div>
            </div>
            <div
              class="flex justify-center p-1 text-md"
            >
              <p
                class="text-center text-wrap break-words"
              />
            </div>
          </div>
          <div
            class="rounded-xl p-2 m-2 bg-slate-700"
          >
            <div
              class="flex justify-center p-2 text-xl"
            >
              <p
                class="text-center text-wrap break-words"
              >
                name2
              </p>
            </div>
            <div
              class="flex justify-center p-2"
            >
              <div
                class="flex rounded-xl m-1"
              >
                <div
                  class="flex p-1 rounded-l-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
                >
                  -
                </div>
                <div
                  class="flex bg-slate-900 p-1 min-w-12 justify-center select-none text-2xl"
                >
                  0
                </div>
                <div
                  class="flex p-1 rounded-r-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
                >
                  +
                </div>
              </div>
            </div>
            <div
              class="flex justify-center p-1 text-md"
            >
              <p
                class="text-center text-wrap break-words"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="text-xl text-white text-center items-center rounded-xl m-1 flex-grow"
    >
      <div
        class="bg-slate-900 rounded-xl m-1 p-2"
      >
        Action Points:
        <br />
        0
        /
        0
      </div>
      <div
        class="grid grid-cols-3 bg-slate-900 rounded-xl m-1 p-2"
      >
        <div
          class="rounded-xl p-2 m-2 bg-slate-700"
        >
          <div
            class="flex justify-center p-2 text-xl"
          >
            <p
              class="text-center text-wrap break-words"
            >
              name
            </p>
          </div>
          <div
            class="flex justify-center p-2"
          >
            <div
              class="flex rounded-xl m-1"
            >
              <div
                class="flex p-1 rounded-l-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
              >
                -
              </div>
              <div
                class="flex bg-slate-900 p-1 min-w-12 justify-center select-none text-2xl"
              >
                0
              </div>
              <div
                class="flex p-1 rounded-r-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
              >
                +
              </div>
            </div>
          </div>
          <div
            class="flex justify-center p-1 text-md"
          >
            <p
              class="text-center text-wrap break-words"
            />
          </div>
        </div>
        <div
          class="rounded-xl p-2 m-2 bg-slate-700"
        >
          <div
            class="flex justify-center p-2 text-xl"
          >
            <p
              class="text-center text-wrap break-words"
            >
              name1
            </p>
          </div>
          <div
            class="flex justify-center p-2"
          >
            <div
              class="flex rounded-xl m-1"
            >
              <div
                class="flex p-1 rounded-l-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
              >
                -
              </div>
              <div
                class="flex bg-slate-900 p-1 min-w-12 justify-center select-none text-2xl"
              >
                0
              </div>
              <div
                class="flex p-1 rounded-r-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
              >
                +
              </div>
            </div>
          </div>
          <div
            class="flex justify-center p-1 text-md"
          >
            <p
              class="text-center text-wrap break-words"
            />
          </div>
        </div>
        <div
          class="rounded-xl p-2 m-2 bg-slate-700"
        >
          <div
            class="flex justify-center p-2 text-xl"
          >
            <p
              class="text-center text-wrap break-words"
            >
              name2
            </p>
          </div>
          <div
            class="flex justify-center p-2"
          >
            <div
              class="flex rounded-xl m-1"
            >
              <div
                class="flex p-1 rounded-l-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
              >
                -
              </div>
              <div
                class="flex bg-slate-900 p-1 min-w-12 justify-center select-none text-2xl"
              >
                0
              </div>
              <div
                class="flex p-1 rounded-r-xl bg-slate-900 min-w-8 justify-center hover:bg-slate-800 hover:cursor-pointer select-none text-2xl"
              >
                +
              </div>
            </div>
          </div>
          <div
            class="flex justify-center p-1 text-md"
          >
            <p
              class="text-center text-wrap break-words"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
